src/indexer-lookup.spec.ts

```diff
@@ -29,16 +29,16 @@ describe('indexer-lookup', () => {
   }, 20_000)
 
   test('Account is found by id', async () => {
-    const { indexer, testAccount, transactionLogger } = localnet.context
-    await transactionLogger.waitForIndexer(indexer)
+    const { indexer, testAccount } = localnet.context
+    await runWhenIndexerCaughtUp(() => algokit.lookupAccountByAddress(testAccount.addr, indexer))
```

Comment:
Not sure why I needed this, but didn't previously. Could mean I'd done something wrong somewhere, or just be part of the evolving mess of indexer performance. I was getting a consistent 404 for this test when running it along with other tests (but not when running it solo). Adding this meant it started working.